crates/cast/tests/e2e/account/create.rs

```diff
@@ -389,3 +389,87 @@ pub fn test_keystore_already_exists(keystore_path: &str, account_path: &str, err
 
     assert!(stderr_str.contains(error));
 }
+
+#[tokio::test]
+pub async fn test_happy_case_keystore_int_format() {
+    let keystore_path = "my_key.json";
+    let account_path = "my_account.json";
+    _ = fs::remove_file(keystore_path);
+    _ = fs::remove_file(account_path);
+    env::set_var(CREATE_KEYSTORE_PASSWORD_ENV_VAR, "123");
+
+    let args = vec![
+        "--url",
+        URL,
+        "--keystore",
+        keystore_path,
+        "--account",
+        account_path,
+        "--int-format",
+        "account",
+        "create",
+        "--class-hash",
+        DEVNET_OZ_CLASS_HASH,
+    ];
+
+    let snapbox = runner(&args);
+
+    snapbox.assert().stdout_matches(indoc! {r#"
+        CREATE_KEYSTORE_PASSWORD environment variable found and will be used for keystore password
+        Account successfully created[..]
+        command: account create
+        add_profile: --add-profile flag was not set. No profile added to Scarb.toml
+        address: [..]
+        max_fee: [..]
```

Comment:
i couldn't find a way to enforce this `[..]` (this wildcard is part of snapbox) to be just numbers. currently this would pass even if those are in format `0x[..]`.

is there anyway to enforce this?

Reply:
not sure tbh, we can leave it like that for now I guess